lib/clickhousex/http_client.ex

```diff
@@ -158,8 +158,11 @@ defmodule Clickhousex.HTTPClient do
     end
   end
 
-  defp maybe_append_format(%Query{type: :select}, request) do
-    [request.post_data, " FORMAT ", Response.format()]
```

Comment:
What if we check only for `FORMAT`? If the user of the library puts `FORMAT JSONEachRow`, this will append also `FORMAT JSONCompact`.

Also, can you add a comment describing why we do this change, as a bigger rework of this library might touch this code again.